### DISCUSSION TITLE
Prompt to Replace Node with Preferred Node for all tables

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5598] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
We have synonymised nodes in our geography tree but when these are selected as the geography on the CO form there is no prompt to keep or change to the preferred value. I was expecting this to work like synonymy in the taxon tree. 

![image](https://user-images.githubusercontent.com/37256050/175990200-f1900c34-401e-4c44-9c56-35dfe10dadcc.png)

**Requested By:** Gillian Brown

**Comment by maxpatiiuk** [comment]
Duplicate of #1223

**Comment by grantfitzsimmons** [comment]
Request is unique, encompasses all tables rather than only determinations.

**Comment by maxpatiiuk** [comment]
I see. Good idea.
Keep in mind those that this feature is not implemented even for the Taxon table in Specify 7

For future reference:
> Implement Determination.PromptToReplaceSynonym remote pref.
>
> That pref is responsible for "Prompt to Replace Taxon with Prefered Taxon" in Specify 6

**Comment by grantfitzsimmons** [comment]
> Hello,
> In Specify 6, within Preferences > Trees, there is an option to enable a 'Prompt to replace Taxon with Preferred Taxon'.
> 
> When enabled, if you select to add a taxon Synonym to a determination in Specify 6 (in single form entry), the system notifies you that you've selected a Synonym and gives you the ability to change your selection to the preferred taxon instead.
> 
> However, this prompt does not appear in Specify 7, should it? It is useful for users to be aware when they are selecting a synonym and be given the option to change to preferred  taxon at that point.

**Requested By:** Zoe W at CSIRO ([Asana](https://app.asana.com/0/1201972579963460/1205290033748588/f))

**Comment by grantfitzsimmons** [comment]
See also https://github.com/specify/specify7/issues/3944
